perses/app/setup_relative_calculation.py

```diff
@@ -73,6 +73,16 @@ def getSetupOptions(filename):
         setup_options['protocol-type'] = 'default'
 
 
+    if 'temperature' not in setup_options:
```

Comment:
im being nitpicky, but maybe it would be best to put some of the defaults into the documentation?

Reply:
That's a good point. I have been working on the documentation in a different branch, so I will update it there